chutney/server/src/main/java/com/chutneytesting/execution/api/CampaignExecutionUiController.java

```diff
@@ -81,9 +82,9 @@ public List<CampaignExecutionReportDto> executeCampaignByName(@PathVariable("cam
         List<CampaignExecution> reports;
         String userId = userService.currentUser().getId();
         if (environment.isPresent()) {
-            reports = campaignExecutionEngine.executeByName(campaignName, environment.get(), userId);
+            reports = campaignExecutionEngine.executeByName(campaignName, environment.get(), null, userId);
         } else {
-            reports = campaignExecutionEngine.executeByName(campaignName, userId);
+            reports = campaignExecutionEngine.executeByName(campaignName, null, null, userId);
```

Comment:
Can't we have some executeByName methods with differents parameters like it was done with environment ?